src/pages/Sketch/components/Canvas.js

```diff
@@ -486,7 +407,7 @@ function Canvas() {
         setFontStyle={setFontStyle}
         fontFamily={fontFamily}
         setFontFamily={setFontFamily}
-        setFillImage={setFillImage}
+        // setFillImage={setFillImage}
```

Comment:
Do not comment out unused code. Remove these in your next PR

Reply:
ok got it